custom/icds_reports/utils/__init__.py

```diff
@@ -1774,3 +1777,81 @@ def create_group_by(aggregation_level):
 
 def column_value_as_per_agg_level(aggregation_level, agg_level_threshold, true_value, false_value):
     return true_value if aggregation_level > agg_level_threshold else false_value
+
+
+@attr.s
+class AggLevelInfo(object):
+    agg_level = attr.ib()
+    col_name = attr.ib()
+
+
+def _construct_repacement_map_from_awc_location(loc_level, replacement_location_ids):
+    levels = {
+        'state': AggLevelInfo(1, 'state'),
+        'district': AggLevelInfo(2, 'district'),
+        'block': AggLevelInfo(3, 'block'),
+        'supervisor': AggLevelInfo(4, 'supervisor'),
+        'awc': AggLevelInfo(5, 'doc')
```

Comment:
nit: it would be nice to put these numbers in constants. I never remember if 1 is state or 1 is AWC but a constant name would be useful.

Reply:
@calellowitz did you think about this?

Reply:
oops, that idea sounds good, and we actually already do have constants for those values. will add them

Reply:
added